cli/include/external-cli/ignition/utils/cli/App.hpp

```diff
@@ -15,4 +15,4 @@
  *
  */
 
-#include <CLI/App.hpp>
+#include <external-cli/gz/utils/cli/App.hpp>
```

Comment:
the `external-cli/` should be removed from the `#include`s in all these files

Reply:
https://github.com/ignitionrobotics/ign-utils/pull/55/commits/f3c7fcdc2ab59e50a8ea105845feaa57b3f221ed